bin/test-workflow/policy/templates/project-authn-def.template.yml

```diff
@@ -36,6 +36,14 @@
         authn-k8s/deployment: test-app-secretless
         authn-k8s/authentication-container-name: secretless
         kubernetes: "{{ IS_KUBERNETES }}"
+    - !host
```

Comment:
We really need to fix this policy to avoid retaining that depricated convention of
`host/conjur/authn-k8s/<authenticator-ID>/<conjur-policy-layer-or-group>/<app-host-id>`. A simple fix would be to split this file in 2
1. For defining the authenticator (without the hosts)
2. For defining the hosts. The hosts could be placed at the root level to avoid the need for the  `CONJUR_AUTHN_LOGIN_PREFIX` envvar

Reply:
Policy loading is going to be reworked as part of #322